### DISCUSSION TITLE
friends/

### DISCUSSION THREAD

**Discussion by giscus** [SHIERTX/static-blog#3] [Comments]
# friends/

https://www.liczhi.com/friends/

**Comment by cheezhi** [comment]
我的友链呢，可否说明原因？

名称：土土博客
介绍：一个初二的渣渣
链接：https://hi-tutu.top
LOGO：https://hi-tutu.top/favorite.png

**Reply by zaochih** [reply]
在逐步迁移啦，目前是照着数据库抄过来的2333

**Reply by cheezhi** [reply]
哦哦，明白啦！

**Reply by zaochih** [reply]
已提交更新，友链已完善~

CF Pages正在缓慢部署中，GH Pages预览在[这里](https://pages.liczhi.com)

**Reply by cheezhi** [reply]
你的博客友链要改成这个吗（liczhi.com），还是十二小栈（https://blog.shiertx.com/）

**Reply by zaochih** [reply]
> 你的博客友链要改成这个吗（liczhi.com），还是十二小栈（https://blog.shiertx.com/）

原来的吧~等我慢慢悠悠迁移完就把那个域名解析过来。

**Comment by cheezhi** [comment]
hi～
土土博客换域名啦
麻烦把友链更改为https://tutime.cn
谢谢～|´・ω・)ノ

**Reply by zaochih** [reply]
咕咕咕咕咕咕咕，周末给你改~
